src/txbuilder/TxBuilder.js

```diff
@@ -39,10 +39,12 @@ import {
     isRight
 } from "@helios-lang/type-utils"
 import { UplcProgramV1, UplcProgramV2, UplcDataValue } from "@helios-lang/uplc"
+import { BasicUplcLogger } from "@helios-lang/uplc/node_modules/@helios-lang/compiler-utils/src/errors/BasicUplcLogger.js"
```

Comment:
I hope it's ok for you that I comment on the remainder of this PR once we've last the *uplc* PR and we can clean lines like this up

Reply:
Oh, that's a mistake.  Sorry about that.

Reply:
was fixed in the later push

Reply:
the import was fixed in the later push, but the symbol isn't used here anymore.